bootstrap.py

```diff
@@ -20,43 +21,44 @@
 with app.app_context():
     session = classic.current_session()
     engine = classic.util.current_engine()
-    logger.info('Waiting for database server to be available')
-    logger.info(app.config['SQLALCHEMY_DATABASE_URI'])
+    logger.info("Waiting for database server to be available")
+    logger.info(app.config["SQLALCHEMY_DATABASE_URI"])
+
     wait = 2
     while True:
         try:
-            session.execute('SELECT 1')
+            session.execute("SELECT 1")
```

Comment:
I think most of the rest of the code base is using single quotes.

Reply:
That's a side effect of running the black formatter on this file with defaults. Here's their reasoning behind [double quotes](https://black.readthedocs.io/en/stable/the_black_code_style.html#strings). We could do something like this if the defaults are annoyingly making things inconsistent:

> If you are adopting Black in a large project with pre-existing string conventions (like the popular “single quotes for data, double quotes for human-readable strings”), you can pass --skip-string-normalization on the command line. This is meant as an adoption helper, avoid using this for new projects.

Reply:
That sounds good to me.